### DISCUSSION TITLE
BIRT 4.9 is released

### DISCUSSION THREAD

**Discussion by wimjongman** [eclipse-birt/birt#872] [General]
BIRT 4.9.0 is released

Downloads: https://download.eclipse.org/birt/downloads/drops/R-R1-4.9.0-202203161719/
Update-site: https://download.eclipse.org/birt/update-site/4.9.0/

Issues resolved for this release:

https://github.com/eclipse/birt/milestone/1?closed=1

**Comment by SteveSchafer-Innovent** [comment]
Is there going to be a birt_4_9_0 tag in github?  Or a release?

**Comment by wimjongman** [comment]
I started to create one but I don't have authority. Maybe we can ask the project lead @TheLaserNinja. I already created branch 4.9.0 which is at the level of the code we have used for the release.

**Reply by TheLaserNinja** [reply]
Weird that it'd require my access to create a tag. You were able to tag the 4.8 release. 

Anyway, I created a release and a tag for it.
